daemon/rpc_claim_online_purchase.go

```diff
@@ -0,0 +1,26 @@
+package daemon
+
+import (
+	"context"
+	"log"
+
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
+	"github.com/NordSecurity/nordvpn-linux/internal"
+)
+
+func (r *RPC) ClaimOnlinePurchase(ctx context.Context, in *pb.Empty) (*pb.Empty, error) {
```

Comment:
this will never return an error. Shouldn't there be one when `isExpired == true` or for `err`?